pmd-custom_ruleset.xml

```diff
@@ -4,25 +4,30 @@
     xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
     xsi:schemaLocation="http://pmd.sf.net/ruleset/1.0.0 http://pmd.sf.net/ruleset_xml_schema.xsd"
     xsi:noNamespaceSchemaLocation="http://pmd.sf.net/ruleset_xml_schema.xsd">
-  <description>
-  Custom PMD checks for TheAlgorithms/Java
-  </description>
-<rule name="UselessMainMethod"
-      language="java"
-      message="The main method is redundant in this context"
-      class="net.sourceforge.pmd.lang.rule.xpath.XPathRule">
+
     <description>
-        Avoid using the main method.
+        Custom PMD checks for TheAlgorithms/Java
```

Comment:
Please revert changes to configuration files and make sure your code passes checks and there are JUnit tests